articles/storage/common/storage-service-encryption.md

```diff
@@ -156,7 +156,7 @@ A: Not at this time; the keys are fully managed by Microsoft.
 
 **Q: Is SSE enabled by default when I create a new storage account?**
 
-A: SSE is not enabled by default; you can use the Azure portal to enable it. You can also programmatically enable this feature using the Storage Resource Provider REST API.
+A: As of August 31, 2017 encryption will be enabled default using Microsoft Managed Keys for all data written to Azure services (Blob, File, Table and Queue storage), for all storage accounts (Azure Resource Manager and Classic storage accounts), both new and existing.
```

Comment:
@JackStromberg - Can you make a couple of grammatical fixes?

As of August 31, 2017, encryption is enabled by default using Microsoft Managed Keys for all data written to Azure Storage (Blob, File, Table and Queue storage), and for all storage accounts (Azure Resource Manager and Classic storage accounts), both new and existing.